lib/terraforming/resource/iam_group_membership.rb

```diff
@@ -44,11 +44,11 @@ def tfstate
       private
 
       def group_members_of(group)
-        @client.get_group(group_name: group.group_name).users.map { |user| user.user_name }
+        @client.get_group(group_name: group.group_name).map(&:users).flatten.map { |user| user.user_name }
```

Comment:
__[rubocop]__
* Line is too long. [93/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Pass `&:user_name` as an argument to `map` instead of a block.